### ISSUE TITLE
Ability to also clear CloudFlare cache

### ISSUE THREAD

**Issue by simplenotezy** [GeekPress/wp-rocket-cli#26]
It would be cool if it could also clear the cloudflare cache

**Comment by alessandro-newzoo** [comment]
agreed, I was hoping `wp rocket clean` would also trigger Varnish, Cloudflare, and whatever else a manual clear from within the Dashboard would trigger.

This would make automatic deployments a lot more enjoyable.
The absence of this functionality renders the command pretty much useless for a lot of users :(
